### PR TITLE
Feat: simple pos

### PR DESCRIPTION
**Pull request by rolznz**
Merges @bumi 's changes to make the PoS simpler with some extra changes from me (see commits).

- Removes the nostr cart
- Inbuilt keyboard
- Moved item label to settings.
- NWC URL is saved in local storage and not included in every page. Share links are encoded in base64.
- Added a temporary redirect from the wallet URL format currently in master.
- Better initial connection handling with Bitcoin Connect (detects payment permission, disconnects from bitcoin connect if permissions are missing)

### CODE REVIEW

**Comment by bumi** [comment]
it was cursor ai's PR mostly :D

**Comment by bumi** [comment]
I would love to use an isolated connection. maybe we can make the permission thing just a warning?

File: src/pages/wallet/New.tsx [hunk 1]
```diff
@@ -67,49 +68,150 @@ export function New() {
...
+                  <option value="CNY">CNY</option>
+                  <option value="RUB">RUB</option>
+                  <option value="INR">INR</option>
+                  <option value="BRL">BRL</option>
+                  <option value="MXN">MXN</option>
+                  <option value="TRY">TRY</option>
+                  <option value="ZAR">ZAR</option>
+                </select>
+              </div>
+            </div>
```
Comment (bumi) [inline src/pages/wallet/New.tsx:152]: you don't want to show the label here?

Comment (rolznz) [reply src/pages/wallet/New.tsx:176]: Re-added

**Pull request merged**